src/core/solvers/ddp.cpp

```diff
@@ -40,13 +41,13 @@ SolverDDP::SolverDDP(boost::shared_ptr<ShootingProblem> problem)
 SolverDDP::~SolverDDP() {}
 
 bool SolverDDP::solve(const std::vector<Eigen::VectorXd>& init_xs, const std::vector<Eigen::VectorXd>& init_us,
-                      const std::size_t& maxiter, const bool& is_feasible, const double& reginit) {
+                      std::size_t maxiter, bool is_feasible, double reginit) {
   xs_try_[0] = problem_->get_x0();  // it is needed in case that init_xs[0] is infeasible
   setCandidate(init_xs, init_us, is_feasible);
 
   if (std::isnan(reginit)) {
-    xreg_ = regmin_;
-    ureg_ = regmin_;
+    xreg_ = reg_min_;
+    ureg_ = reg_min_;
```

Comment:
I remember seeing regInit=1e-9 in many solvers. The logic that we have in this file should be copied to the other solvers (I think it was in fddp that this default value is set, among others)